rust-sdk/src/client.rs

```diff
@@ -394,8 +352,196 @@ impl<'a> Client<'a> {
         Ok(Some(convert(value)))
     }
 
+    /// Get the assignment value for a given feature flag and subject, along with details of why
+    /// this value was selected.
+    ///
+    /// *NOTE:* It is a debug function and is slower due to the need to collect all the
```

Comment:
Is it really that much slower? Our JS implementation always calls it, and we haven't heard any complaints (at least not yet).

I almost think in a different world, if we started from scratch, we'd _only_ have this method.

Reply:
Yes, see above comment